mkdocs/docs/HPC/VScode.md

```diff
@@ -0,0 +1,46 @@
+# VS Code Tunnel
+
+Please keep in mind, that this is not a VS Code manual,
+it is only our recommendation how to connect to the {{ hpcinfra }} using the VS code tunnel. 
+
+For VS Code documentation, see <https://code.visualstudio.com/docs>
+
+We only support VS Code tunnel access via the VS Code application (installed locally),
+since web access might not work from all browser or operational system.
+The only confirmed way to connect via a browser is using Windows 11 and Edge. 
+
+## Prerequisites
+
+We do recommend to install remote developent for the VS Code App.
+
+
+## Connection
+
+In the [web portal](./web_portal.md), under the "Interactive Apps" choose VS Code Tunnel,
+select the parameters and launch your job by clicking ++"Launch"++
+When your job has started, connect by clicking ++"Connect"++
+
+In the new browser tab or window, for the question
+`How would you like to log in to Visual Studio Code?` choose `Microsoft Account` 
+(using the arrow keys on your keyboard, then hit ++"Enter"++).
+Then follow the instruction to authenticate yourself.
+If you get an error message something like "You do not have access to this"
+durng the Microsoft Authentication, then you might have to
+[whitelist yourself](https://dictselfservice.ugent.be/index.php?page=requestform&form=deviceCodeAuth)
+(You have to do it only once.)
```

Comment:
add that this is only for ugent users and they can use `Need VS Code tunnel for HPC` as motivation text